clang/lib/Format/Format.cpp

```diff
@@ -1448,6 +1464,7 @@ FormatStyle getLLVMStyle(FormatStyle::LanguageKind Language) {
   LLVMStyle.AlignConsecutiveAssignments.PadOperators = true;
   LLVMStyle.AlignConsecutiveBitFields = {};
   LLVMStyle.AlignConsecutiveDeclarations = {};
+  LLVMStyle.AlignConsecutiveDeclarations.AlignFunctionDeclarations = true;
```

Comment:
I assume this is true by default because it was true at line 1023 (thats ok just wanted to check)

Reply:
right, so the default behavior previously was true.  The new option technically just allows it to be set to false.  So we're keeping the prior behavior by default.